export_dir/junk.yaml

```diff
@@ -0,0 +1,63 @@
+description: Butler Data Repository Export
```

Comment:
(Already mentioned this above, but why are we versioning something called `junk.yaml`?)